src/components/CreateRules/UploadRules.vue

```diff
@@ -1,54 +1,208 @@
+            </template>
+            <span>Tooltip text goes here.  text is TBD.</span>
+          </v-tooltip>.
+        </p>
+      </header>
+
+    </section>
+    <!-- Help Section -->
+    <div v-if="getCreateRulesResource.helpSection" class="mt-5">
+      <span class="help-btn" @click="helpToggle = !helpToggle">
+        <v-icon color="blue darken-2" style="padding-right: 5px">mdi-help-circle-outline</v-icon>
+        <span v-if="!helpToggle">{{ getCreateRulesResource.helpSection.header }}</span>
+        <span v-else>Hide Help</span>
+      </span>
+      <section v-show="helpToggle" class="create-rules-help">
+        <header id="create-rules-help-header"><h2>{{getCreateRulesResource.helpSection.header}}</h2></header>
+        <p>
+          <b>{{getCreateRulesResource.helpSection.helpText.section1.label}}</b>
+        </p>
+        <ul>
+          <li
+            v-for="(item, index) in getCreateRulesResource.helpSection.helpText.section1.items"
+            class="help-section"
+            :key="index"
+          >{{ item }}
+          </li>
+        </ul>
+        <br/>
+        <p>
+          <b>{{getCreateRulesResource.helpSection.helpText.section2.label}}</b>
+        </p>
+        <ul>
+          <li
+            v-for="(item, index) in getCreateRulesResource.helpSection.helpText.section2.items"
+            class="help-section"
+            :key="index"
+          ><span v-html="item"></span>
+          </li>
+        </ul>
+        <br/>
+        <p
+          v-for="(item, index) in getCreateRulesResource.helpSection.helpText.section3.items"
+          class="help-section"
+          :key="index"
+        >{{ item }}
+        </p>
+
+        <u class="help-btn" @click="helpToggle = !helpToggle"><small>Hide Help</small></u>
+      </section>
     </div>
 
-    <v-card-title>Rules Placeholder component</v-card-title>
-  </v-card>
+    <section class="mt-10">
+      <header>
+        <h2>2. Upload Completed Rules</h2>
+        <p>
+          <v-checkbox
+            id="chk-confirm-rules"
+            v-model="rulesConfirmed"
+            label="I confirm the following items are included as required in the Rules of the Association."
+            @change="onRulesConfirmedChange()"
+          />
+          <ul>
+            <li>The cooperative name is identified <b>exactly</b> as followings in the rules:</li>
+            <p><b>{{getNameRequestDetails.approvedName}}</b></p>
+            <li>Each Subscriber and Witness has signed and dated the Rules of the Association and their name is printed
+              under their signature.</li>
+          </ul>
+        </p>
+      </header>
+    </section>
+    <v-card flat id="upload-rules">
+      <v-row>
+        <v-col cols="2" >
+          <v-card-title style="padding-top:1px;font-size:medium;">Upload Rules</v-card-title>
+        </v-col>
+        <v-col cols="9" >
+          <FileUploadPreview
+            :maxSize="MAX_FILE_SIZE"
+            v-bind:input-file="uploadRulesDoc"
```

Comment:
updated to use `:inputFile="{...uploadRulesDoc}"`

Reply:
Oops, I didn't mean "destructure".

I mean to change
`v-bind:input-file="uploadRulesDoc"`
to
`:input-file="uploadRulesDoc"`.

(The ... was me trying to avoid typing what followed v-bind ;) )

Reply:
ah...haha.  ok, will fix.